src/revlanguage/analysis/mcmc/output/RlBurninEstimationConvergenceAssessment.cpp

```diff
@@ -161,9 +157,9 @@ RevPtr<RevVariable> BurninEstimationConvergenceAssessment::executeMethod(std::st
         for (size_t p = 0; p < vectorOfFileNames.size(); p++)
         {
             
-            const std::string &fn = vectorOfFileNames[p];
+            auto &fn = vectorOfFileNames[p];
             
-            RBOUT("\tProcessing file '" + fn + "'");
+            RBOUT("\tProcessing file '" + fn.string() + "'");
```

Comment:
Hmm.... it seems inconsistent that RBOUT is not a stream like std::cout.
I think we should fix RBOUT so that you could write `RBOUT<<"Processing file "<<fn`.
But perhaps not in this PR, which is large enough already...

Reply:
Hmm....I think the bigger question is whether you can add an operator+(string,T) for every T.
I tend to think that that is a bad idea.